voila/app.py

```diff
@@ -241,6 +241,11 @@ class Voila(Application):
                                  cannot be determined reliably by the Jupyter notebook server (proxified
                                  or containerized setups for example)."""))
 
+    prelaunch_hook = Any(default_value=None, allow_none=True,
```

Comment:
Instead of `Any` here, maybe we should use `Instance(Callable)`.

`Callable` coming from the builtin `typing` module https://docs.python.org/3/library/typing.html#typing.Callable

Reply:
changed